OleksiiBozhchenko/HW7/Task3.py

```diff
@@ -0,0 +1,11 @@
+# Write a function that calculate thr number of characters included in given string.
+# input: "hello"
+# output: {"h":1,"e":1,"l":2,"o":1}
+
```

Comment:
here is a more optimal implementation:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result